internal/webhooks/machine_webhook.go

```diff
@@ -14,9 +14,10 @@ See the License for the specific language governing permissions and
 limitations under the License.
```

Comment:
I would suggest to drop the _webhook suffix from these files as we're already in the webhooks package